addon-test-support/@ember/test-helpers/dom/tab.ts

```diff
@@ -0,0 +1,248 @@
+import getRootElement from './get-root-element';
+import settled from '../settled';
+import fireEvent, { buildKeyboardEvent } from './fire-event';
+import { isDocument } from './-target';
+import { __blur__ } from './blur';
+import { nextTickPromise } from '../-utils';
+import { createTreeWalker } from './-create-tree-walker';
+import { __focus__ } from './focus';
+
+const FORM_TAGS = ['INPUT', 'BUTTON', 'SELECT', 'TEXTAREA', 'FIELDSET'];
```

Comment:
```suggestion
const FORM_TAGS = ['INPUT', 'BUTTON', 'SELECT', 'TEXTAREA'];
```
A fieldset isn't one of the interactive elements for form tags (assuming that's what you're going for here).

Reply:
That part is actually handled by UA, with the `tabIndex` property of the element. The `tabIndex` property returns default values, or the `tabindex` attribute, parsed as a number. It doesn’t take into account anything else though, so we’ll need to manually handle that. One of these cases are the disabled attribute, this variable is the list of tags that can be disabled. When walking the DOM, it rejects any disabled elements and its children. 

It means it won’t tab to this input element in this case:

```html
<fieldset disabled>
  <input />
</fieldset>
```

The name of the variable doesn’t really describe that, will come up with a better name.